lib.rs

```diff
@@ -464,6 +463,78 @@ impl ThreadPool {
     }
 }
 
+impl Clone for ThreadPool {
+    /// Cloning a pool will create a new handle to the pool.
+    /// The behavior is similar to [Arc](https://doc.rust-lang.org/stable/std/sync/struct.Arc.html).
+    ///
+    /// We could for example [`join`](#method.join) a pool from multiple
+    /// threads or submit jobs from multiple threads concurrently.
+    ///
+    /// ```
```

Comment:
No worries 😄 
I could shorten the code by half by using a loop to spawn the threads and then use another loop/fold to join them. Or would you go into a completely different direction?